components/widgets/users/avatar.jsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './avatar.scss';
+
+const Avatar = ({url, username, size}) => (
+    <img
+        className={'Avatar ' + size}
+        alt={`${username || 'user'} profile image`}
+        src={url}
+    />
+);
+
+Avatar.propTypes = {
+    url: PropTypes.string.isRequired,
+    username: PropTypes.string,
+    size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl', 'xxl']),
+};
+
+Avatar.defaultProps = {
+    size: 'md',
```

Comment:
Can we have `avatar-md` etc as the class?
Also, is there a reason why we're going with a capital A for the class?
Similar to how we have `col-md` or `col-xl`.
I would prefer the classes to be similar and descriptive.